File: models/Package.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const packageSchema = new Schema({
    name: {
        type: String,
        required: [true, 'Please enter a name']
    },
    price: {
        type: Number,
        required: [true, 'Please enter a price']
    },
    sessionDiscount: {
        type: Number,
        required: [true, 'Please enter a session discount']
    },
    medicineDiscount: {
        type: Number,
        required: [true, 'Please enter a medicine discount']
    },
    familyDiscount: {
        type: Number,
        required: [true, 'Please enter a family discount']
    },
}, {timestamps: true});

const Package = mongoose.model('Package', packageSchema, 'Package');
module.exports = Package;

File: clinic_front_end/lib/Models/Package.dart
class Package {
  String name = "";
  double price = 0;
  double sessionDiscount = 0;
  double medicalDiscount = 0;
  double familyDiscount = 0;

  Package.fromJson(Map<String, dynamic> json) {
    name = json['Name'];
    price = json['Price'];
    sessionDiscount = json['SessionDiscount'];
    medicalDiscount = json['MedicineDiscount'];
    familyDiscount = json['FamilyDiscount'];
  }
}
